File: tests/GeneralFixture/java/FiveFixtureAttributes/src/FooTest.java
import org.junit.TestCase;

public class FooTest extends TestCase {

    Bar bar;
    Baz baz;
    Fal fal;
    Foo foo;
    Fuu fuu;

    public void setUp() {
        bar = new Bar();
        baz = new Baz();
        fal = new Fal();
        foo = new Foo();
        fuu = new Fuu();
    }

    public void testFoo() {
        assertTrue(foo.foo());
    }
}

File: tests/Pieces/GeneralFixture/cpp/FiveFixtureAttributes/src/FooTest.cpp
#include "FooTest.h"

void FooTest::setUp() 
{
	bar = new Bar;
	baz = new Baz;
	fal = new Fal;
	foo = new Foo;
	fuu = new Fuu;
}

void FooTest::testFoo() 
{
	CPPUNIT_ASSERT(true);
}
